src/main/java/com/epam/ta/reportportal/core/launch/DeleteLaunchHandler.java

```diff
@@ -51,4 +52,15 @@ OperationCompletionRS deleteLaunch(Long launchId, ReportPortalUser.ProjectDetail
    */
   DeleteBulkRS deleteLaunches(DeleteBulkRQ deleteBulkRQ,
       ReportPortalUser.ProjectDetails projectDetails, ReportPortalUser user);
+
+  /**
+   * Bulk launches delete.
+   *
+   * @param ids   list of launch ids
+   * @param projectDetails Project Details
+   * @param user           User
+   * @return DeleteLaunchesRS
+   */
+  DeleteBulkRS deleteLaunches(List<Long> ids,
```

Comment:
interface implementation class should be updated accordingly

Reply:
DeleteBulkRS - response of api don't change